### ISSUE TITLE
Error while training starts. How to resolve?

### ISSUE THREAD

**Issue by PRYUS** [ooleksyuk/CarND-Semantic-Segmentation#2]
Segmentation fault (core dumped) comes when the first epoch starts.
I guess the error is in 
for image, label in tqdm(generator, total=n_train, desc=description, unit='batches'):
            _, loss, _ = sess.run([train_op, cross_entropy_loss, iou_op],
                                  feed_dict={input_image: image, correct_label: label,
                                             keep_prob: KEEP_PROB, learning_rate: lr})

in main_city.py
